packages/terra-toolbar/src/Toolbar.jsx

```diff
@@ -12,6 +12,24 @@ const propTypes = {
    * Content alignment. Align to start, end, or center.
    */
   align: PropTypes.oneOf(['start', 'end', 'center']),
+  /**
+   * String that labels the content the toolbar controls. Should reference
+   * the id of the content area that the toolbar controls.
+   */
+  ariaControls: PropTypes.string,
+  /**
+   * String that labels the content the toolbar controls. `ariaLabel` only be used
+   * when no visible label for the toolbar is present. `ariaLabelledBy` should be
+   * used over `ariaLabel` when possible.
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * String that labels the content the toolbar controls. Should reference
```

Comment:
```suggestion
   * String that labels the content of the toolbar controls. Should reference
```

Reply:
Updated the wording for clarity [d92fdde](https://github.com/cerner/terra-core/pull/3678/commits/d92fdde1469050de5ac17f97563b115a8d4162a4)